python/templates/Constructor.jl.jinja2

```diff
@@ -0,0 +1,26 @@
+include("{{ class.bare_type }}Struct.jl")
+{% for include in includes_jl['constructor'] %}
+{{ include }}
+{% endfor %}
+function {{ class.bare_type }}()
+	return {{ class.bare_type }}{% if params_jl %}{ {% for relation in params_jl %} {{ relation }}, {% endfor %} }{% endif %}(
+	{% for member in Members %}
+	{% if member.is_array %}
+	{{ member.julia_type }}(undef),
```

Comment:
Can we initialize this with another default (that makes sense)?

Reply:
we can initialize it using other methods but, undef is better due to [better perfmance](https://stackoverflow.com/questions/58430581/julia-what-is-undef-in-array-in-julia) as it just allocates the memory required instead of initializing each variable.

Reply:
That is a very good point. I didn't think about that.